scripts/frontmatter-validator/frontmatter-validator.js

```diff
@@ -114,38 +144,35 @@ function validateFiles(fileListPath) {
 function main() {
   const args = process.argv.slice(2);
 
-  if (args.includes("--file")) {
-    const fileListIndex = args.indexOf("--file") + 1;
+  if (args.includes('--file')) {
+    const fileListIndex = args.indexOf('--file') + 1;
     const fileListPath = args[fileListIndex];
     validateFiles(fileListPath);
   } else {
     const filesChanged = args[0];
     if (filesChanged) {
       const filePaths = filesChanged
-        .split(",")
-        .filter((file) => file.endsWith(".md"))
+        .split(',')
+        .filter((file) => file.endsWith('.md'))
         .map((file) => `../../${file}`);
       filePaths.forEach(validateFrontmatter);
     }
   }
 
   if (allErrors.length === 0) {
-    console.log("No frontmatter validation errors found.");
+    console.log('No frontmatter validation errors found.');
     return;
   }
 
   allErrors.forEach(({ filePath, fileErrors }) => {
     console.log(
       `## Rule: [${filePath}](https://github.com/SSWConsulting/SSW.Rules.Content/tree/main/${filePath})\n`
     );
-    console.log("Issues:");
+    console.log('Issues:');
```

Comment:
Maybe instead of just Issues, it could be `Issues Detected:`

Reply:
Kind of like this? If so done.

![image](https://github.com/user-attachments/assets/78c5fafc-35e2-481d-bf25-e1338b2773c0)

Reply:
Amazing 🚀